File: kyw0716/2주차/첫 시도/이상한 문자 만들기.js
function solution(s) {
  return s
    .split(" ")
    .map((word) => {
      return word
        .split("")
        .map((alphabet, index) => {
          if (index % 2 == 0) {
            return alphabet.toUpperCase();
          }
          return alphabet.toLowerCase();
        })
        .join("");
    })
    .join(" ");
}
// https://school.programmers.co.kr/learn/courses/30/lessons/12930

File: sk000801/LEVEL1/이상한 문자 만들기.java
class Solution {
    public String solution(String s) {
        String[] answer = s.split("");
        String real = "";   
        int count = 0;
        for(String ss : answer) {
            count = ss.contains(" ") ? 0 : count+1 ;
            real += count%2 == 0 ? ss.toLowerCase() : ss.toUpperCase();
        }
        return real;
    }
}
